src/common/src/array/data_chunk_iter.rs

```diff
@@ -250,6 +250,19 @@ impl Row {
         &EMPTY_ROW
     }
 
+    /// Compare two rows' stream key
+    pub fn eq_by_pk(
+        row1: impl AsRef<Self>,
+        pk_indices1: &[usize],
+        row2: impl AsRef<Self>,
+        pk_indices2: &[usize],
+    ) -> bool {
```

Comment:
```suggestion
    pub fn eq_by_key(
        row1: impl AsRef<Self>,
        key1: &[usize],
        row2: impl AsRef<Self>,
        key2: &[usize],
    ) -> bool {
```

Reply:
Will fix in the next pr

Reply:
Suggest fixing it in this PR. Also better not to mention the term of "stream key" in the **data chunk** module.